packages/sources/src/styling/variables/media-queries-breakpoints.scss

```diff
@@ -1,5 +1,5 @@
 // Extra small screen / phone
-$screen-xs-min: 480px !default; //** Deprecated `$screen-xs-min` as of v3.2.0, only used in _navbar.scss
+$screen-xs-min: 480px !default; //** Deprecated `$screen-xs-min` as of v3.2.0, only used in _navbar.scss & card.scss
```

Comment:
Beetje suf dat deze comment is meegekomen naar `sources`, maar nog suffer dat we 'm blijven uitbreiden. In `sources` is niets deprecated. Wat mij betreft haal je de hele comment weg (alsof `_navbar.scss` nog bestaat in `sources`).